docs/tutorials/edc-cut-bell-pair-benchmarking.ipynb

```diff
@@ -0,0 +1,715 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9857bace",
+   "metadata": {},
+   "source": [
+    "{/* cspell:ignore LOCC */}\n",
+    "{/* cspell:ignore bitstr */}\n",
+    "{/* cspell:ignore textcoords */}\n",
+    "{/* cspell:ignore xytext */}\n",
+    "{/* cspell:ignore fontsize */}\n",
+    "{/* cspell:ignore  Carrera */}\n",
+    "{/* cspell:ignore  Ristè */}\n",
+    "# Benchmark dynamic circuits with cut Bell pairs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d08a5e51",
+   "metadata": {},
+   "source": [
+    "*Estimated QPU usage: 22 seconds (tested on IBM Kingston)*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "77dca78b",
+   "metadata": {},
+   "source": [
+    "## Background\n",
+    "\n",
+    "Quantum hardware is typically limited to local interactions, but many algorithms require entangling distant qubits or even [qubits on separate processors](#references). Dynamic circuits, that is circuits with mid-circuit measurement and feedforward, provide a way to overcome these limitations by using real-time classical communication to effectively implement non-local quantum operations. In this approach, measurement outcomes from one part of a circuit (or one QPU) can conditionally trigger gates on another, allowing us to teleport entanglement across long distances. This forms the basis of **Local Operations and Classical Communication (LOCC)** schemes, where we consume entangled resource states (Bell pairs) and communicate measurement results classically to link far-apart qubits.\n",
+    "\n",
+    "One promising use of LOCC is to realize virtual long-range CNOT gates by teleportation as shown in the [long-range entanglement tutorial](https://quantum.cloud.ibm.com/docs/en/tutorials/long-range-entanglement). Instead of a direct long-range CNOT (which hardware connectivity may not permit), we create Bell pairs and perform a teleportation-based gate implementation. However, the fidelity of such operations depends on hardware characteristics. Qubit decoherence during the necessary delay (while waiting for measurement results) and classical communication latency can degrade the entangled state. Also, errors on mid-circuit measurements are harder to correct than errors on final measurements as they propagate to the rest of the circuit through the conditional gates.\n",
```

Comment:
```suggestion
    "One promising use of LOCC is to realize virtual long-range CNOT gates by teleportation, as shown in the [long-range entanglement tutorial](/docs/tutorials/long-range-entanglement). Instead of a direct long-range CNOT (which hardware connectivity might not permit), we create Bell pairs and perform a teleportation-based gate implementation. However, the fidelity of such operations depends on hardware characteristics. Qubit decoherence during the necessary delay (while waiting for measurement results) and classical communication latency can degrade the entangled state. Also, errors on mid-circuit measurements are harder to correct than errors on final measurements as they propagate to the rest of the circuit through the conditional gates.\n",
```